src/cmd/rio/winwatch.c

```diff
@@ -36,14 +37,13 @@ int showwmnames;
 Font *font;
 Image *lightblue;
 
-XErrorHandler oldxerrorhandler;
 
-enum {
+enum 
+{
```

Comment:
So this is the sort of formatting change I'm talking about; it obscures the main intent of the change with a lot of extraneous noise. If these parts of the patch could be reverted, that'd be great.